qutip/tests/test_wigner.py

```diff
@@ -593,6 +595,21 @@ def test_wigner_clenshaw_sp_iter_dm():
         Wdiff = abs(W - Wclen)
         assert_equal(np.sum(abs(Wdiff)) < 1e-7, True)
 
-
+@pytest.mark.slow
```

Comment:
You can parameterise the spin with:
```python
@pytest.mark.parametrize(['spin'], [
    pytest.param(1/2, id="spin-one-half"),
    pytest.param(3, id="spin-three"),
    ...
])
```

Reply:
Question: How slow is slow?

Reply:
If it is very slow, perhaps we can just reduce the number of steps down from 32 to say, 7?

Reply:
I've tested with pytest and the `spin_q_function` and `spin_wigner` tests together take less than 1 minute to complete.

Thanks for the tip on parametrisation, I've also added an arg to test non-pure state.